static/app/views/feedback/list.tsx

```diff
@@ -31,18 +42,43 @@ export default function List() {
       <PageFiltersContainer>
         <Layout.Body>
           <Layout.Main fullWidth>
-            {isLoading ? (
-              <Placeholder />
-            ) : isError ? (
-              <Alert type="error" showIcon>
-                {t('An error occurred')}
-              </Alert>
-            ) : (
-              <pre>{JSON.stringify(data, null, '\t')}</pre>
-            )}
+            <LayoutGap>
+              <PageFilterBar condensed>
+                <ProjectPageFilter resetParamsOnChange={['cursor']} />
+                <EnvironmentPageFilter resetParamsOnChange={['cursor']} />
+                <DatePageFilter alignDropdown="left" resetParamsOnChange={['cursor']} />
+              </PageFilterBar>
+              <FeedbackTable
+                data={data ?? []}
+                isError={isError}
+                isLoading={isLoading}
+                location={location}
+              />
+            </LayoutGap>
+            <PaginationNoMargin
+              pageLinks={
+                pageLinks ||
+                '<https://sentry.sentry.io/api/0/organizations/sentry/replays/?field=activity&field=browser&field=count_dead_clicks&field=count_errors&field=count_rage_clicks&field=duration&field=finished_at&field=id&field=is_archived&field=os&field=project_id&field=started_at&field=urls&field=user&per_page=50&project=11276&query=&sort=-started_at&statsPeriod=24h&cursor=0:0:1>; rel="previous"; results="false"; cursor="0:0:1", <https://sentry.sentry.io/api/0/organizations/sentry/replays/?field=activity&field=browser&field=count_dead_clicks&field=count_errors&field=count_rage_clicks&field=duration&field=finished_at&field=id&field=is_archived&field=os&field=project_id&field=started_at&field=urls&field=user&per_page=50&project=11276&query=&sort=-started_at&statsPeriod=24h&cursor=0:50:0>; rel="next"; results="true"; cursor="0:50:0"'
```

Comment:
Should this be removed? or maybe mark with `TODO` if we need to keep it temporarily

Reply:
ya, removed